### PR TITLE
Minor code updates to create_right_hand_side

### PR DESCRIPTION
**Pull request by gfcas**
Minor updates on the code.

### CODE REVIEW

**Review by drwells — APPROVED** [review]
👍

File: include/deal.II/numerics/vector_tools_rhs.templates.h [hunk 1]
```diff
@@ -378,7 +364,7 @@ namespace VectorTools
       {
```
Comment (drwells) [inline include/deal.II/numerics/vector_tools_rhs.templates.h:364]: You need to get rid of the now-unused `cell` and `endc` defined on lines 359-361.

Comment (gfcas) [reply include/deal.II/numerics/vector_tools_rhs.templates.h:360]: Thanks for this hint. Now it is fixed.

**Pull request merged**